qt/applications/workbench/workbench/widgets/settings/presenter.py

```diff
@@ -12,26 +12,35 @@
 from mantid import ConfigService
 from mantidqt.interfacemanager import InterfaceManager
 from workbench.widgets.settings.categories.presenter import CategoriesSettings
+from workbench.widgets.settings.fitting.presenter import FittingSettings
 from workbench.widgets.settings.general.presenter import GeneralSettings
+from workbench.widgets.settings.plots.presenter import PlotSettings
 from workbench.widgets.settings.view import SettingsView
 
 
 class SettingsPresenter(object):
     ASK_BEFORE_CLOSE_TITLE = "Confirm exit"
     ASK_BEFORE_CLOSE_MESSAGE = "Are you sure you want to exit without applying the settings?"
     SETTINGS_TABS = {'general_settings' : "General",
-                     'categories_settings' : "Show/Hide Categories"}
+                     'categories_settings' : "Show/Hide Categories",
```

Comment:
I know this is not strictly this PR but I think I missed the other one. It looks like "Show/Hide Categories" is always too wide for the width of the left-hand side box. Should we shorten this to just `Categories` to be consistent with the other single words? Perhaps we could add the Show /Hide to the "Algorithm Categories" & "Interface Categories" titles?